<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarTrailer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="filesList">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>10</number>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
      <widget class="View" name="graphicsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_directory"/>
    <addaction name="action_Save_as"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLive_preview_while_composite">
     <property name="title">
      <string>Live preview while composite</string>
     </property>
     <addaction name="actionWithout_preview"/>
     <addaction name="actionPreviewEach_1s"/>
     <addaction name="actionPreviewEach_2s"/>
     <addaction name="actionPreviewEach_5s"/>
     <addaction name="actionPreviewEach_10s"/>
     <addaction name="actionPreviewEach_15s"/>
     <addaction name="actionPreviewEach_30s"/>
     <addaction name="actionPreviewEach_60s"/>
    </widget>
    <addaction name="actionAuto_trail_on_select"/>
    <addaction name="menuLive_preview_while_composite"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuComposite_mode">
     <property name="title">
      <string>Composite mode</string>
     </property>
     <addaction name="actionAdd"/>
     <addaction name="actionAtop"/>
     <addaction name="actionBlend"/>
     <addaction name="actionBumpmap"/>
     <addaction name="actionClear"/>
     <addaction name="actionColorize"/>
     <addaction name="actionCopy"/>
     <addaction name="actionCopyBlack"/>
     <addaction name="actionCopyBlue"/>
     <addaction name="actionCopyCyan"/>
     <addaction name="actionCopyGreen"/>
     <addaction name="actionCopyMagenta"/>
     <addaction name="actionCopyOpacity"/>
     <addaction name="actionCopyRed"/>
     <addaction name="actionCopyYellow"/>
     <addaction name="actionDArken"/>
     <addaction name="actionDifference"/>
     <addaction name="actionDisplace"/>
     <addaction name="actionDissolve"/>
     <addaction name="actionDstOver"/>
     <addaction name="actionExclusion"/>
     <addaction name="actionHardLight"/>
     <addaction name="actionHue"/>
     <addaction name="actionIn"/>
     <addaction name="actionLighten"/>
     <addaction name="actionLuminize"/>
     <addaction name="actionMinus"/>
     <addaction name="actionModulate"/>
     <addaction name="actionMyltiply"/>
     <addaction name="actionNo"/>
     <addaction name="actionOut"/>
     <addaction name="actionOver"/>
     <addaction name="actionOverlay"/>
     <addaction name="actionPlus"/>
     <addaction name="actionSaturate"/>
     <addaction name="actionScreen"/>
     <addaction name="actionSoftLight"/>
     <addaction name="actionSubtract"/>
     <addaction name="actionThreshold"/>
     <addaction name="actionXor"/>
    </widget>
    <widget class="QMenu" name="menuRAW_processing_mode">
     <property name="title">
      <string>RAW processing mode</string>
     </property>
     <addaction name="actionOnly_jpeg_preview"/>
     <addaction name="actionLibraw_half_size"/>
     <addaction name="actionLibraw_full_size"/>
    </widget>
    <addaction name="menuComposite_mode"/>
    <addaction name="menuRAW_processing_mode"/>
    <addaction name="actionComposite_2"/>
    <addaction name="actionCopy_to_clipboard"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Selection</string>
    </property>
    <addaction name="actionClear_2"/>
    <addaction name="actionAll"/>
    <addaction name="separator"/>
    <addaction name="actionSelectEach_2"/>
    <addaction name="actionSelectEach_5th"/>
    <addaction name="actionSelectEach_10th"/>
    <addaction name="actionSelectEach_25th"/>
    <addaction name="actionSelectEach_50th"/>
    <addaction name="actionSelectEach_100"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuSelection"/>
   <addaction name="menuImage"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionUpdateFileList"/>
   <addaction name="actionComposite"/>
   <addaction name="actionClearSelection"/>
   <addaction name="actionPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionComposite">
   <property name="text">
    <string>Composite</string>
   </property>
  </action>
  <action name="actionUpdateFileList">
   <property name="text">
    <string>Update file list</string>
   </property>
  </action>
  <action name="actionClearSelection">
   <property name="text">
    <string>Clear Selection</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>_</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionLighenIntencity">
   <property name="text">
    <string>LighenIntencity</string>
   </property>
  </action>
  <action name="actionOnly_jpeg_preview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Only jpeg preview</string>
   </property>
  </action>
  <action name="actionLibraw_half_size">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Libraw half_size dcraw_process()</string>
   </property>
  </action>
  <action name="actionLibraw_full_size">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Libraw full size dcraw_process()</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionAtop">
   <property name="text">
    <string>Atop</string>
   </property>
  </action>
  <action name="actionBlend">
   <property name="text">
    <string>Blend</string>
   </property>
  </action>
  <action name="actionBumpmap">
   <property name="text">
    <string>Bumpmap</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionColorize">
   <property name="text">
    <string>Colorize</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCopyBlack">
   <property name="text">
    <string>CopyBlack</string>
   </property>
  </action>
  <action name="actionCopyBlue">
   <property name="text">
    <string>CopyBlue</string>
   </property>
  </action>
  <action name="actionCopyCyan">
   <property name="text">
    <string>CopyCyan</string>
   </property>
  </action>
  <action name="actionCopyGreen">
   <property name="text">
    <string>CopyGreen</string>
   </property>
  </action>
  <action name="actionCopyMagenta">
   <property name="text">
    <string>CopyMagenta</string>
   </property>
  </action>
  <action name="actionCopyOpacity">
   <property name="text">
    <string>CopyOpacity</string>
   </property>
  </action>
  <action name="actionCopyRed">
   <property name="text">
    <string>CopyRed</string>
   </property>
  </action>
  <action name="actionCopyYellow">
   <property name="text">
    <string>CopyYellow</string>
   </property>
  </action>
  <action name="actionDArken">
   <property name="text">
    <string>Darken</string>
   </property>
  </action>
  <action name="actionDifference">
   <property name="text">
    <string>Difference</string>
   </property>
  </action>
  <action name="actionDisplace">
   <property name="text">
    <string>Displace</string>
   </property>
  </action>
  <action name="actionDissolve">
   <property name="text">
    <string>Dissolve</string>
   </property>
  </action>
  <action name="actionDstOver">
   <property name="text">
    <string>DstOver</string>
   </property>
  </action>
  <action name="actionExclusion">
   <property name="text">
    <string>Exclusion</string>
   </property>
  </action>
  <action name="actionHardLight">
   <property name="text">
    <string>HardLight</string>
   </property>
  </action>
  <action name="actionHue">
   <property name="text">
    <string>Hue</string>
   </property>
  </action>
  <action name="actionIn">
   <property name="text">
    <string>In</string>
   </property>
  </action>
  <action name="actionLighten">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lighten</string>
   </property>
  </action>
  <action name="actionLuminize">
   <property name="text">
    <string>Luminize</string>
   </property>
  </action>
  <action name="actionMinus">
   <property name="text">
    <string>Minus</string>
   </property>
  </action>
  <action name="actionModulate">
   <property name="text">
    <string>Modulate</string>
   </property>
  </action>
  <action name="actionMyltiply">
   <property name="text">
    <string>Myltiply</string>
   </property>
  </action>
  <action name="actionNo">
   <property name="text">
    <string>No</string>
   </property>
  </action>
  <action name="actionOut">
   <property name="text">
    <string>Out</string>
   </property>
  </action>
  <action name="actionOver">
   <property name="text">
    <string>Over</string>
   </property>
  </action>
  <action name="actionOverlay">
   <property name="text">
    <string>Overlay</string>
   </property>
  </action>
  <action name="actionPlus">
   <property name="text">
    <string>Plus</string>
   </property>
  </action>
  <action name="actionSaturate">
   <property name="text">
    <string>Saturate</string>
   </property>
  </action>
  <action name="actionScreen">
   <property name="text">
    <string>Screen</string>
   </property>
  </action>
  <action name="actionSoftLight">
   <property name="text">
    <string>SoftLight</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="text">
    <string>Subtract</string>
   </property>
  </action>
  <action name="actionThreshold">
   <property name="text">
    <string>Threshold</string>
   </property>
  </action>
  <action name="actionXor">
   <property name="text">
    <string>Xor</string>
   </property>
  </action>
  <action name="actionAuto_trail_on_select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto trail on select</string>
   </property>
  </action>
  <action name="actionClear_2">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionSelectEach_2">
   <property name="text">
    <string>Each 2nd</string>
   </property>
  </action>
  <action name="actionSelectEach_5th">
   <property name="text">
    <string>Each 5th</string>
   </property>
  </action>
  <action name="actionSelectEach_10th">
   <property name="text">
    <string>Each 10th</string>
   </property>
  </action>
  <action name="actionSelectEach_25th">
   <property name="text">
    <string>Each 25th</string>
   </property>
  </action>
  <action name="actionSelectEach_50th">
   <property name="text">
    <string>Each 50th</string>
   </property>
  </action>
  <action name="actionSelectEach_100">
   <property name="text">
    <string>Each 100</string>
   </property>
  </action>
  <action name="actionPreviewEach_1s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 1 second</string>
   </property>
  </action>
  <action name="actionPreviewEach_2s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 2 seconds</string>
   </property>
  </action>
  <action name="actionPreviewEach_5s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 5 seconds</string>
   </property>
  </action>
  <action name="actionPreviewEach_10s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 10 seconds</string>
   </property>
  </action>
  <action name="actionPreviewEach_15s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 15 seconds</string>
   </property>
  </action>
  <action name="actionPreviewEach_30s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 30 seconds</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionWithout_preview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Without preview</string>
   </property>
  </action>
  <action name="actionComposite_2">
   <property name="text">
    <string>Composite</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play (time lapse)</string>
   </property>
  </action>
  <action name="actionPreviewEach_60s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Each 60 seconds</string>
   </property>
  </action>
  <action name="actionOpen_directory">
   <property name="text">
    <string>Open directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCopy_to_clipboard">
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionComposite_2</sender>
   <signal>triggered()</signal>
   <receiver>actionComposite</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
